File: ios/uclass/uclass/network/response/SNSCheckData.swift
/**
 * SNS 체크/로그인 응답 데이터 모델
 */
struct SNSCheckData: Codable {
    let isExistingUser: Bool
    let redirectUrl : String?
    
    private enum CodingKeys: String, CodingKey {
        case isExistingUser = "existingUser"
        case redirectUrl = "redirectUrl"
    }
}

File: android/UClass/app/src/main/java/com/ubase/uclass/network/response/SNSCheckData.kt
package com.ubase.uclass.network.response

import com.google.gson.annotations.SerializedName

/**
 * SNS 체크 응답 데이터 모델
 */
data class SNSCheckData(
    @SerializedName("existingUser")
    val isExistingUser: Boolean,

    @SerializedName("redirectUrl")
    val redirectUrl: String
)